ci/properties/cmake-multi-platform.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "CMake based multi-platform projects",
```

Comment:
```suggestion
    "name": "CMake based, multi-platform projects",
```

Reply:
Will make this change and push the updates.